compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -1311,6 +1311,23 @@ impl<'a> PartialEvaluator<'a> {
         args_span: PackageSpan,        // For diagnostic purposes only.
         callee_expr_span: PackageSpan, // For diagnostic puprposes only.
```

Comment:
not your fault but typo in comment here if we are taking drive by suggestions

Reply:
Fixed!